defaults/main.yml

```diff
@@ -122,5 +122,13 @@ auth_ldap_system_users:
     gid: 1000
     initialPassword: "{{ auth_kerberos_ldap_password }}"
 
+auth_ldap_user_objectclasses:
```

Comment:
Can you add a comment here that `inetUser` is added automatically when the `memberOf` overlay is enabled?